### PR TITLE
Empty boxes on GTFS-RT

### PR DESCRIPTION
**Pull request by pbougue**
Beware for later work: working on gtfs_rt branch so far (even on master)

### CODE REVIEW

File: kirin/gtfs_rt/model_maker.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+class KirinModelBuilder(object):
```
Comment (pbougue) [inline kirin/gtfs_rt/model_maker.py]: Maybe more GtfsRTKirinModelBuilder (and rename Ire's one also, then).
Any opinion @antoine-de @kinnou02 ?

**Comment by antoine-de** [comment]
Jenkins, test this please

File: kirin/gtfs_rt/gtfs_rt.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            rt_update.error = e.data['error']
+            model.db.session.add(rt_update)
+            model.db.session.commit()
+            raise
+        except Exception as e:
+            rt_update.status = 'KO'
+            rt_update.error = e.message
+            model.db.session.add(rt_update)
+            model.db.session.commit()
+            raise
```
Comment (xlqian) [inline kirin/gtfs_rt/gtfs_rt.py]:
```python
try:
    trip_updates = KirinModelBuilder(self.navitia_wrapper, self.contributor).build(rt_update)
except KirinException as e:
    rt_update.error = e.data['error']
except Exception as e:
    rt_update.error = e.message
finally:
    rt_update.status = 'KO'
    model.db.session.add(rt_update)
    model.db.session.commit()
    raise
```

Comment (pbougue) [reply kirin/gtfs_rt/gtfs_rt.py]: It would be done even without exception, so not the same behaviour

Comment (xlqian) [reply kirin/gtfs_rt/gtfs_rt.py]: my bad, but at least we can do a little refactoring here

Comment (xlqian) [reply kirin/gtfs_rt/gtfs_rt.py]:
```python
try:
    trip_updates = KirinModelBuilder(self.navitia_wrapper, self.contributor).build(rt_update)
except KirinException as e:
    rt_update.error = e.data['error']
except Exception as e:
    rt_update.error = e.message
finally:
    if rt_update.error:
        rt_update.status = 'KO'
        model.db.session.add(rt_update)
        model.db.session.commit()
        raise
```

**Pull request merged**